dune-project

```diff
@@ -5,6 +5,79 @@
 (formatting
  (enabled_for ocaml))
 
-; TODO
-;
-; (generate_opam_files true)
+(generate_opam_files true)
+
+(license LGPL-2.1)
+(maintainers "The Coq development team <coqdev@inria.fr>")
+(authors "The Coq development team, INRIA, CNRS, and contributors")
+; This generates bug-reports and dev-repo
+(source (github coq/coq))
+(homepage https://coq.inria.fr/)
+(documentation "https://coq.github.io/doc/")
+(version dev)
+
+; Note that we use coq.opam.template to have dune add the correct opam
+; prefix for configure
+(package
+ (name coq)
+ (depends
+  (ocaml (>= 4.05.0))
+  (dune  (>= 2.5.0))
+  (ocamlfind :build)
```

Comment:
I thought build only dependencies were not something considered good anymore.

Reply:
It depends, as far as I now they are not frowned upon, in this case ocamlfind is only used in configure so it would be OK.

Oh wait, actually nativelib does call ocamlfind so this is not a build-only dep.